mlir/test/Dialect/Linalg/canonicalize.mlir

```diff
@@ -1096,3 +1096,30 @@ func.func @transpose_transpose_fold(%input: tensor<5x4x3xf32>,
   func.return %transpose2 : tensor<3x4x5xf32>
 }
 
+
+// -----
+
+func.func @concats_of_fill(
```

Comment:
For static shapes, it would be nice to replace with a single `empty` + `fill`. But this could be part of another cleanup?

Reply:
That would happen with this https://github.com/llvm/llvm-project/commit/c077a4f305aa7faf92a1438b239078c1da1563a9